aggregator/log_manager.py

```diff
@@ -0,0 +1,45 @@
+# Python interface for managing aggregated logs
+
+import sqlite3
+
+class LogManager:
+
+  def __init__(self):
+    self.conn = sqlite3.connect('aggregated_logs.db')
+    with open('aggregator.sql', 'rb') as initialization_file:
+      self.conn.executescript(initialization_file.read())
+    self.cursor = self.conn.cursor()
+
+  # Adds log entry into database
+  #
+  # params:
+  #   log_entry: tab-separated column values for log 
+  def add_log_entry(self, log_entry):
+    log_columns = log_entry.split("\t")
+    if len(log_columns) == 7:
+      self.cursor.execute('INSERT OR REPLACE INTO Log VALUES (?, ?, ?, ?, ?, ?, ?)',
```

Comment:
@paivaspol I put that in to make sure we don't have duplicate tuple errors. (e.g. you query the same server multiple times for the log from the same day)

The primary key on Log is on `(timestamp, uuid, source_entity, destination_entity)`, so this should take care of uniqueness.

Thoughts?